src/Raven.Studio/wwwroot/Content/css/pages/storage-report.less

```diff
@@ -437,4 +448,28 @@
             margin-left: -4px;
         }
     }
+
+    .th-min-width {
+        min-width: 200px;
+    }
+
+    .th-limited-width {
+        width: 180px;
+    }
+
+    .table-items {
+        position: absolute;
+        display: flex;
+        width: 100%;
+        padding-right: 20px;
+    }
+
+    .table-item-name {
+        display: block;
+        white-space: nowrap;
+    }
+}
+
+.position-relative {
```

Comment:
can we avoid having global scoped classes in view specific file?

Reply:
Fixed.